cmd/kube-controller-manager/app/options/options.go

```diff
@@ -66,6 +66,7 @@ type CMServer struct {
 
 	ClusterName       string
 	ClusterCIDR       net.IPNet
+	PodCIDRMaskSize   int
```

Comment:
It looks like you started to make pod CIDR sizes configurable, but then didn't use it anywhere?